dns/bind/pkg-descr

```diff
@@ -8,6 +8,10 @@ necessary for asking and answering name service questions.
 Plugin Changelog
 ================
 
+1.5
+
+* Add basic zone management
```

Comment:
still "basic" ?

Reply:
Let's leave it basic. There is stuff to improve, but I'd say it depends how many ppl will give feedback (Views and PTR missing, some automations could be added and so on)